schemas/top-api.yml

```diff
@@ -397,6 +375,36 @@ paths:
           $ref: '#/components/responses/UnexpectedError'
       tags:
         - conceptPipeline
+    put:
+      operationId: startConceptGraphPipelineWithJson
+      summary: |
+        Initialises a complete pipeline with all steps 'preprocessing', 'embedding', 'clustering' and 'graph creation'.
+        Doesn't support file uploading; configuration is done via json body.
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/ConceptPipelineConfigRequest'
+      responses:
+        '200':
+          description: A short summary of the graphs created by this pipeline.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/PipelineResponse'
+        '202':
+          description: The PUT request was accepted and a pipeline process started.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/PipelineResponse'
+        '401':
+          $ref: '#/components/responses/Unauthorized'
+        default:
+          $ref: '#/components/responses/UnexpectedError'
+      tags:
+        - conceptPipeline
```

Comment:
I'm not really sure/pleased about/with this, but there is no way (as far as I found) to have multiple `content`s in a `requestBody`